spec/controllers/idv/image_uploads_controller_spec.rb

```diff
@@ -111,6 +155,31 @@
                                remaining_attempts: 0,
                              })
         end
+
+        it 'tracks analytics' do
+          allow(Throttler::IsThrottledElseIncrement).to receive(:call).once.and_return(true)
+          allow(Throttler::RemainingCount).to receive(:call).and_return(0)
+
+          stub_analytics
+
+          expect(@analytics).to receive(:track_event).with(
+            Analytics::IDV_DOC_AUTH_SUBMITTED_FORM,
+            {
+              success: false,
+              errors: {
+                limit: [I18n.t('errors.doc_auth.acuant_throttle')],
+              },
+              remaining_attempts: 0,
+            },
+          )
```

Comment:
ditto for this re: hash arguments. Also if there is a particular hash key/value we care about, we could do `hash_including` to just highlight the one key (instead of having this big assertion in every single spec) for example

```suggestion
          expect(@analytics).to receive(:track_event).with(
            Analytics::IDV_DOC_AUTH_SUBMITTED_FORM,
            hash_including(remaining_attempts: 0),
          )
```

Reply:
> Also if there is a particular hash key/value we care about, we could do `hash_including` to just highlight the one key (instead of having this big assertion in every single spec) for example

I'd actually been thinking about this in relation to the presenter we added in #4185, where part of the purpose of the presenter was to avoid including additional properties in the response that we hadn't been explicitly anticipating (see https://github.com/18F/identity-idp/pull/4185#issuecomment-691190585). While I'd grant it's more verbose, if we'd been exhaustive in detailing all of the [expected properties of the response JSON](https://github.com/18F/identity-idp/blob/dde621350bf8e1ffe0d9f7cae75fedd0db886264/spec/controllers/idv/image_uploads_controller_spec.rb#L46-L49), those unanticipated properties wouldn't have made their way through. Similarly, these assertions here prevent us from including properties we're not expecting. That being said, it probably isn't as big a problem for analytics as it is for our public API responses.

Reply:
> ditto for this re: hash arguments.

Updated in 02a57da.